jest.setup.js

```diff
@@ -1,6 +1,3 @@
-/* eslint-disable no-console */
-import { setLogger } from 'react-query';
```

Comment:
removed setLogger

https://tanstack.com/query/v4/docs/guides/migrating-to-react-query-4#setlogger-is-removed

```js
const queryClient = new QueryClient({ logger: customLogger })
```